src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/Configuration.php

```diff
@@ -45,6 +45,7 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
                 ->booleanNode('intercept_redirects')->defaultFalse()->end()
+                ->booleanNode('enable_cors')->defaultFalse()->end()
```

Comment:
Oh, I just realized I read your comment to fast :) Of course, it's better to enable it by default, I'm gonna update this